index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ro-RO">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Milka homepage</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Lobster&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+</head>
+<body>
+    <div class="nav-wrap">
+        <label id="hamburger-menu">
+            <input type="checkbox"/>
+        </label>
+        <aside class="sliding-menu">
+            <nav>
+                <a href="produse.html">Produse</a>
+                <a href="despre.html">Despre Milka</a>
+                <a href="#">Sustenabilitate</a>
+                <a href="#">Povestea Milka</a>
+            </nav>
+        </aside>
+        <a href="index.html"> <img src="/Proiect-HTML-CSS/images/milka-logo-purple.webp" alt="logo Milka" style="max-width: 30vw;"></a>

```

Comment:
Mai bine pui adrese relative, care să nu conțină path-ul care apare în.

```suggestion
        <a href="index.html"> <img src="images/milka-logo-purple.webp" alt="logo Milka" style="max-width: 30vw;"></a>
```